readme.txt

```diff
@@ -62,6 +62,9 @@ Not at the moment. We will roll out GiveWP recurring donations addon support in
 Yes. It does support GiveWP fee recovery addon
 
 == Changelog ==
+= ✅ 1.0.1 =
+Add Support to Wordpress 6.7
```

Comment:
@Ankur0812 change text to "Added support for WordPress 6.7"

Reply:
Changed Text from Add to Added.